code/modules/disposals/disposal_chute.dm

```diff
@@ -268,6 +306,15 @@ ADMIN_INTERACT_PROCS(/obj/machinery/disposal, proc/flush, proc/eject)
 		else
 			return ..()
 
+	proc/set_broken()
+		if(src.status & BROKEN)
+			return
+		src.eject()
+		ZERO_GASES(src.air_contents)
```

Comment:
i feel like it'd expel the air into the surroundings instead of magically deleting it